lib/jxl/decode.cc

```diff
@@ -1887,12 +1887,11 @@ JXL_EXPORT JxlDecoderStatus JxlDecoderImageOutBufferSize(
   JxlDecoderStatus status = PrepareSizeCheck(dec, format, &bits);
   if (status != JXL_DEC_SUCCESS) return status;
 
-  size_t row_size =
-      jxl::DivCeil(dec->metadata.size.xsize() * format->num_channels * bits,
-                   jxl::kBitsPerByte);
+  size_t row_size = dec->metadata.size.xsize();
   if (format->align > 1) {
-    row_size = jxl::DivCeil(row_size, format->align) * format->align;
+    row_size = jxl::RoundUpTo(row_size, format->align);
```

Comment:
the user documentation at `JxlPixelFormat` in types.h says that the alignment is in bytes. It seems to perform the alignment before multiplying with num_channels here. Does this match the API documentation now?